inst/include/Rcpp/exceptions.h

```diff
@@ -127,17 +127,17 @@ namespace Rcpp{
 } // namespace Rcpp
 
 inline SEXP get_last_call(){
-    SEXP sys_calls_symbol = Rf_install( "sys.calls" ) ;
-    Rcpp::Shield<SEXP> sys_calls_expr( Rf_lang1(sys_calls_symbol) );
+    SEXP sys_calls_symbol = Rf_install( "sys.call" ) ;
+
+    // -9 Skips the wrapped tryCatch from Rcpp_eval
+    Rcpp::Shield<SEXP> sys_calls_expr( Rf_lang2(sys_calls_symbol, Rf_ScalarInteger(-9)) );
```

Comment:
https://github.com/RcppCore/Rcpp/pull/582/commits/bb82bd94e34caab8a5390514ba38f2aae1e194c6 now matches the expression exactly rather than relying on the position in the call stack